types/interval_year_month_type.go

```diff
@@ -58,3 +58,7 @@ func (IntervalYearToMonthType) ShortString() string { return "intrytm" }
 func (m IntervalYearToMonthType) String() string {
 	return fmt.Sprintf("intervalyeartomonth%s", strNullable(m))
 }
+
+func (m IntervalYearToMonthType) Match(ot Type) bool {
+	return m == ot
```

Comment:
delegate to `m.Equals(ot)`?

Reply:
not needed since API changed to "MatchWithNullability" and "MatchWithoutNullability"